Makefile

```
@@ -192,3 +193,12 @@ scheduler-plugins-chart: scheduler-plugins-crd
 	cp -f $(PROJECT_DIR)/dep-crds/scheduler-plugins/crd.yaml $(PROJECT_DIR)/dep-manifests/scheduler-plugins/crds/scheduling.x-k8s.io_podgroups.yaml
 	cp -f /tmp/pkg/mod/sigs.k8s.io/scheduler-plugins@$(SCHEDULER_PLUGINS_VERSION)/manifests/noderesourcetopology/crd.yaml $(PROJECT_DIR)/dep-manifests/scheduler-plugins/crds/topology.node.k8s.io_noderesourcetopologies.yaml
 	chmod -R 760 $(PROJECT_DIR)/dep-manifests/scheduler-plugins
+
+.PHONY: volcano-scheduler
+volcano-scheduler:
+	-@GOPATH=/tmp go install volcano.sh/volcano/cmd/scheduler@$(VOLCANO_SCHEDULER_VERSION)
+
+.PHONY: volcano-scheduler-crd
+volcano-scheduler-crd: volcano-scheduler
+	mkdir -p $(PROJECT_DIR)/dep-crds/volcano-scheduler/
+	cp -f /tmp/pkg/mod/volcano.sh/volcano@$(VOLCANO_SCHEDULER_VERSION)/config/crd/bases/* $(PROJECT_DIR)/dep-crds/volcano-scheduler
```

Using a cached repository would be good like this:

```suggestion
VOLCANO_ROOT = $(shell go list -m -f "{{.Dir}}" volcano.sh/volcano)
.PHONY: volcano-scheduler-crd
volcano-scheduler-crd:
	mkdir -p $(shell pwd)/dep-crds/volcano-scheduler/
	cp -f $(VOLCANO_ROOT)/config/crd/bases/*  $(shell pwd)/dep-crds/volcano-scheduler/
```

---

I copied it from `make scheduler-plugins-crd` command. Should we keep same style or change `make scheduler-plugins-crd` also?

---

If it works for scheduler-plugins, I prefer to apply the same way to the scheduler-plugins.